content/collections/audit-works.html

```diff
@@ -1,3 +1,262 @@
+<item>
+  <field name="logo">
+    sommelier.png
+  </field>
+  <field name="title">
+    Sommelier-12
+  </field>
+  <field name="subtitle">
+    Automated DeFi yield optimization strategies
+  </field>
+  <field name="date">
+    2023-10-27
+  </field>
+  <field name="about">
```

Comment:
Audited the new Aura position adaptor, Curve and Redstone pricing extensions, and small updates to the Frax adaptors.